rancher/v2.0/en/quick-start-guide/index.md

```diff
@@ -4,91 +4,248 @@ layout: rancher-default-v2.0
 version: v2.0
 lang: en
 redirect_from:
-  - /rancher/quick-start-guide/
-  - /rancher/v2.0/en/
+ /rancher/quick-start-guide/
+ /rancher/v2.0/en/
```

Comment:
when you run this locally, do the redirects still work?

Reply:
They do. Here's what happened: I installed a package in atom that beautifies markdown on save. Unfortunately, this package "beautified" the header as well--it added a bunch of garbage characters to the redirects so they weren't working. The update you see here is what I had after removing all the garbage characters. I tested them and they work.